src/clj/owlet/routes/contentful.clj

```diff
@@ -179,7 +179,8 @@
             @(http/put (subscriber-endpoint id)
                        {:body (json/encode
                                 {:email (:email subscriber)
-                                 :confirmed (not confirmed?)})})]
+                                 :confirmed (when-not (= nil confirmed?)
```

Comment:
Hi @ZadenRB , this works. Nice catch. FYI,  Clojure also has a `nil?` predicate function built in :)
```
(when-not (nil? confirmed)
```